user-guide/Advanced_Modules/DOM/Tutorials/DOM_Remove_Enum_Entry.md

```diff
@@ -0,0 +1,406 @@
+            _engine = engine;
+            _domHelper = new DomHelper(_engine.SendSLNetMessages, ModuleName);
+            
+            // Create the ModuleSettingsHelper
+            var moduleSettingsHelper = new ModuleSettingsHelper(_engine.SendSLNetMessages);
+
+            // Create the default moduleSettings and save it
+            var moduleSettings = new ModuleSettings(ModuleName);
+            moduleSettingsHelper.ModuleSettings.Create(moduleSettings);
+
+            // Start creating the DOM objects now that you have the domHelper
+            var firstNameFieldDescriptor = new FieldDescriptor()
+            {
+                FieldType = typeof(string),
+                IsOptional = false,
+                Name = "First name",
+                Tooltip = "First name of the person"
+            };
+
+            var lastNameFieldDescriptor = new FieldDescriptor()
+            {
+                FieldType = typeof(string),
+                IsOptional = false,
+                Name = "Last name",
+                Tooltip = "Last name of the person"
+            };
+
+            var phoneNumberFieldDescriptor = new FieldDescriptor()
+            {
+                FieldType = typeof(string),
+                IsOptional = false,
+                Name = "Phone number",
+                Tooltip = "Phone number of the person"
+            };
+
+            var genericEnum = new GenericEnum<int>();
+            genericEnum.AddEntry("Director", 0);
+            genericEnum.AddEntry("Project manager", 1);
+            genericEnum.AddEntry("Engineer", 2);
+            genericEnum.AddEntry("Secretary", 3);
+
+            var jobTitleFieldDescriptor = new GenericEnumFieldDescriptor()
+            {
+                FieldType = typeof(GenericEnum<int>),
+                IsOptional = true,
+                GenericEnumInstance = genericEnum,
+                Name = "JobTitle",
+                Tooltip = "Job title of the person"
+            };
+
+            // Put all the fieldDescriptors in a sectionDefinition
+            var sectionDefinition = new CustomSectionDefinition()
+            {
+                Name = "PeopleSectionDefinition"
+            };
+
+            sectionDefinition.AddOrReplaceFieldDescriptor(firstNameFieldDescriptor);
+            sectionDefinition.AddOrReplaceFieldDescriptor(lastNameFieldDescriptor);
+            sectionDefinition.AddOrReplaceFieldDescriptor(phoneNumberFieldDescriptor);
+            sectionDefinition.AddOrReplaceFieldDescriptor(jobTitleFieldDescriptor);
+
+            sectionDefinition = _domHelper.SectionDefinitions.Create(sectionDefinition) as CustomSectionDefinition;
+
+            // Create the domDefinition
+            var personDomDefinition = new DomDefinition()
+            {
+                Name = "PersonDomDefinition",
+                SectionDefinitionLinks = new List<SectionDefinitionLink>()
+                {
+                    new SectionDefinitionLink(sectionDefinition?.GetID())
```

Comment:
We can ignore the null check here '?'. That should never be the case, and if it were, the script should stop anyway. So I feel they can be removed on lines 129 and 135.